providers/gce/gce_subnetworks.go

```diff
@@ -27,11 +28,12 @@ func newSubnetworkMetricContext(request, region string) *metricContext {
 }
 
 // GetSubnetwork returns the GCE resource for the compute.Subnetwork if it exists.
-func (g *Cloud) GetSubnetwork(projectName, region, subnetworkName string) (*compute.Subnetwork, error) {
+func (g *Cloud) GetSubnetwork(region, subnetworkName string) (*compute.Subnetwork, error) {
 	ctx, cancel := cloud.ContextWithCallTimeout()
 	defer cancel()
 
 	mc := newSubnetworkMetricContext("get", region)
-	subnetwork, err := g.service.Subnetworks.Get(projectName, region, subnetworkName).Context(ctx).Do()
```

Comment:
using the service directly breaks test mocks as it calls the implementation directly instead of the interface